action.sh

```diff
@@ -2,8 +2,11 @@
 
 set -euxo pipefail
 
-mkdir -p $2
-for path in $1/*.toml; do
-  filename=$(basename $path)
-  toml-to-ical $path -o $2/${filename%.*}.ics
+mkdir -p "$2"
+
+shopt -s globstar
+for path in "$1"/**/*.toml; do
+  filepath=$(realpath $path --relative-base=$1)
```

Comment:
When I try this locally with `$1 = .` (which is the default value from `action.yml` then it fails:

```
❯ bash ./s.sh . result
+ mkdir -p result
+ shopt -s globstar
+ for path in "$1"/**/*.toml
++ realpath ./_timezones.toml --relative-base=.
realpath: --relative-base=.: No such file or directory
+ filepath=/Users/davidtw/Development/calendar/_timezones.toml
```

Reply:
I didn't run `toml-to-ical-bin` at all, I just had it echo the command it would have run and `realpath` failed in trying to do that.